v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1626 @@
+   "outputs": [],
+   "source": [
+    "# from azureml.pipeline.core import StepRun\n",
+    "# ! pip show azureml-pipeline-core"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.contrib.automl.pipeline.steps.utilities import get_output_from_mm_pipeline\n",
+    "\n",
+    "forecasting_results_name = \"forecasting_results\"\n",
+    "forecasting_output_name = \"many_models_inference_output\"\n",
+    "forecast_file = get_output_from_mm_pipeline(\n",
+    "    inference_run, forecasting_results_name, forecasting_output_name, output_file_name\n",
+    ")\n",
+    "df = pd.read_csv(forecast_file)\n",
+    "print(\n",
+    "    \"Prediction has \", df.shape[0], \" rows. Here the first 5 rows are being displayed.\"\n",
+    ")\n",
+    "df.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.rename(columns={\"Predictions\": \"predicted\"}, inplace=True)\n",
+    "df.to_csv(os.path.join(OUTPUT_DIR, \"test-set-predictions-many-models.csv\"), index=False)\n",
+    "df.head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.3 Inference the baseline model\n",
+    "\n",
+    "Next, we perform a rolling evaluation on the test set for the baseline model. To do this, we use the `run_remote_inference` method which downloads the pickle file of the model into the temporary folder `forecast_naive` and copies the `inference_script_naive.py` file to it. This folder is then uploaded on the compute cluster where inference is performed. The `inference_script_naive.py` script performs a rolling evaluation on the test set, similarly to what we have done in section 6.2. Upon completion of this step, we delete the newly created `forecast_naive` folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "baseline_run = remote_base_run.get_best_child()\n",
+    "baseline_model_name = baseline_run.properties[\"model_name\"]\n",
+    "baseline_run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "from scripts.helper_scripts import run_remote_inference_naive\n",
+    "\n",
+    "if True:\n",
+    "    remote_base_run_test = run_remote_inference_naive(\n",
+    "        test_experiment=test_experiment_base,\n",
+    "        compute_target=compute_target,\n",
+    "        train_run=baseline_run,\n",
+    "        test_dataset=test_dataset,\n",
+    "        target_column_name=target_column_name,\n",
+    "        rolling_evaluation_step_size=forecast_horizon,\n",
+    "        inference_folder=\"./forecast_naive\",  # needed otherwise it will be looking for DNN environment b/c/ model.pt is uploaded to the clsuter for inference\n",
```

Comment:
Remove the comment: # needed otherwise it will be looking for DNN environment b/c/ model.pt is uploaded to the clsuter for inference\n",
 
Add a suggestion, <Ctrl+g>
Add heading textAdd bold text, <Ctrl+b>Add italic text, <Ctrl+i>
Add a quote, <Ctrl+Shift+.>Add code, <Ctrl+e>Add a link, <Ctrl+k>
Add a bulleted list, <Ctrl+Shift+8>Add a numbered list, <Ctrl+Shift+7>Add a task list, <Ctrl+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply

Reply:
done